Example/Core/Tests/FIRAppTest.m

```diff
@@ -671,14 +670,13 @@ - (void)testAnalyticsNotSetByGlobalDataCollectionSwitch {
 
   id configurationMock = OCMClassMock([FIRAnalyticsConfiguration class]);
   OCMStub([configurationMock sharedInstance]).andReturn(configurationMock);
-  OCMStub([configurationMock setAnalyticsCollectionEnabled:OCMOCK_ANY persistSetting:OCMOCK_ANY]);
 
   // Reject any changes to Analytics when the data collection changes.
+  OCMReject([configurationMock setAnalyticsCollectionEnabled:YES persistSetting:NO]);
```

Comment:
Do we also want to reject `persistSetting:YES` here just in case anything else changes?